Rules/UseConsistentWhitespace.cs

```diff
@@ -364,6 +375,47 @@ private IEnumerable<DiagnosticRecord> FindOpenParenViolations(TokenOperations to
             }
         }
 
+        private IEnumerable<DiagnosticRecord> FindParameterViolations(Ast ast)
+        {
+            var commandAsts = ast.FindAll(
+                    testAst => testAst is CommandAst, true);
+            foreach (CommandAst commandAst in commandAsts)
+            {
+                var commandParameterAstElements = commandAst.FindAll(testAst => true, searchNestedScriptBlocks: false).ToList();
+                for (int i = 0; i < commandParameterAstElements.Count - 1; i++)
+                {
+                    var leftExtent = commandParameterAstElements[i].Extent;
+                    var rightExtent = commandParameterAstElements[i + 1].Extent;
+                    var extentsAreOnSameLine = leftExtent.EndLineNumber == rightExtent.StartLineNumber;
+                    if (!extentsAreOnSameLine)
+                    {
+                        continue;
+                    }
+                    var expectedStartColumnNumberOfRightExtent = leftExtent.EndColumnNumber + 1;
+                    if (rightExtent.StartColumnNumber > expectedStartColumnNumberOfRightExtent)
+                    {
+                        int numberOfRedundantWhiteSpaces = rightExtent.StartColumnNumber - expectedStartColumnNumberOfRightExtent;
+                        var correction = new CorrectionExtent(
+                            leftExtent.StartLineNumber,
+                            leftExtent.EndLineNumber,
+                            leftExtent.EndColumnNumber + 1,
+                            leftExtent.EndColumnNumber + 1 + numberOfRedundantWhiteSpaces,
+                            string.Empty,
```

Comment:
Worth naming this parameter

Reply:
OK, I even named all parametes at the end